### DISCUSSION TITLE
Add advlist plugin in tinyMCE editor

### DISCUSSION THREAD

**Discussion by felixvo** [directus/directus#13397] [Archive]
We're using the HTML text field heavily in our system.
Would be great if we could have [advlist](https://www.tiny.cloud/docs/plugins/opensource/advlist/) in the tinyMCE editor

![image](https://user-images.githubusercontent.com/24991670/169196079-03ccfafb-ae7f-47d2-a902-672234c834d3.png)

**Comment by azrikahar** [comment]
This has been brought up in #12327 as well, and ideally the current interface can be updated to load/configure plugins as described by this answer here: https://github.com/directus/directus/discussions/12327#discussioncomment-2423262 👍
